modules/oe_showcase_list_pages/config/install/facets.facet.oelp_oe_sc_news__publication_date.yml

```
@@ -1,13 +1,13 @@
 langcode: en
 status: true
 dependencies: {  }
-id: publication_date
+id: oelp_oe_sc_news__publication_date
```

Actually, change this too.
```suggestion
id: oelp_oe_sc_news__oe_publication_date
```